packages/perseus/src/widgets/__tests__/radio/base-radio_test.jsx

```diff
@@ -14,9 +14,7 @@ import BaseRadio from "../../radio/base-radio.jsx";
 import type {APIOptions} from "../../../types.js";
 
 function renderBaseRadio(props) {
-    const apiOptions: APIOptions = {
-        styling: {radioStyleVersion: "final"},
-    };
+    const apiOptions: APIOptions = Object.freeze({});
```

Comment:
Flow was being weird with an empty object, so I used a solution from: https://stackoverflow.com/questions/46840089/object-literal-inexact-type-is-incompatible-with-exact-type-no-object-spread

Reply:
Yes, Flow is weird.  If you add at least one property to the object then the `Object.freeze()` call isn't necessary. 🤷‍♂️

Reply:
We use `Object.freeze()` all over the place to get around this Flow-ism.